docs/tutorial/managing-ontology-project.md

```diff
@@ -0,0 +1,54 @@
+# Tutorial on Managing OBO Ontology Projects
+
+This tutorial is not about editing ontologies and managing the evolution of its content (aka ontology curation), but the general process of managing an ontology project overall. In this lesson, we will cover the following:
+
+1. How to effectively manage an ontology project using GitHub projects and teams
+2. How to coordinate the evolution of ontologies across projects and grants
+
+## Roles in OBO Ontology project management activities
+
+1. *Ontology Curators*: manage the content of ontologies and interact with users
+1. *Principal Ontology Curators*: coordinate the curation activities and have _always_ fixed hours assigned to the project.
```

Comment:
Maybe Editor instead of curator. Consistent with GO terminology, also with terminology within EBI.  No big deal though.

Reply:
Done.